services/docker-compose.yml

```diff
@@ -269,9 +266,9 @@ services:
     networks:
       - default
       - computational_services_subnet
-  #--------------------------------------------------------------------
+
   postgres:
-    image: postgres:10
+    image: postgres:10.10
```

Comment:
v11 is out for a while. shall we upgrade? maybe we open an issue?

Reply:
I just pinned the version we had "to the last digit"